frontend/packages/dev-console/src/components/monitoring/alerts/monitoring-alerts-utils.tsx

```diff
@@ -0,0 +1,116 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { expandable, sortable, cellWidth } from '@patternfly/react-table';
+import {
+  Rule,
+  Alert,
+  StateCounts,
+  Severity,
+  AlertState,
+} from '@console/internal/components/monitoring';
+import { Kebab } from '@console/internal/components/utils';
+import { alertDescription, alertState } from '@console/internal/reducers/monitoring';
+
+const viewAlertRule = {
+  label: 'View Alerting Rule',
+};
+
+enum AlertStates {
+  firing = 'Firing',
+  pending = 'Pending',
+  silenced = 'Silenced',
+}
+
+enum Severities {
```

Comment:
Should reuse the existing `AlertSeverity` enum here. (Also, these severity strings are actually lower case.)

Reply:
Done.